assets/src/components/index.js

```diff
@@ -0,0 +1 @@
+export { default as EventSelect } from './selection/event-select';
```

Comment:
is there any chance that using "default" will cause name collisions?
or is it standard practice to export things named as "default" because the name gets changed at some other point?
what happens if one needed to utilize two different components that were both exported using the "default" name?

Reply:
`default` is used here because I'm pulling in the default export from `./selection/event-select`  So this is essentially saying, export the default from the given file's default export and alias it to `EventSelect` for any files importing this.  

This is a standard way to pass through exports.

Reply:
omg... somehow I read `as` incorrectly... can't even remember what I thought it was, 
but ya, what you said is painfully obvious now :facepalm:  sry about that